src/extension/extension.ts

```diff
@@ -271,7 +272,25 @@ export async function activate(context: vs.ExtensionContext, isRestart: boolean
 			};
 		}
 
-		flutterDaemon = new FlutterDaemon(logger, workspaceContext as FlutterWorkspaceContext, flutterCapabilities, runIfNoDevices);
+		if (workspaceContext.config.forceFlutterWorkspace) {
+			let resultFromLocalExtension = null;
+			try {
+				resultFromLocalExtension = await vs.commands.executeCommand<string>("dartlocaldevice.startDaemon", workspaceContext.config.flutterToolsScript?.script, "expose_devices", workspaceContext.config.flutterSdkHome);
```

Comment:
It may be worth making these args an object (`[{ flutterToolsScript: xxx, flutterSdkHome: xxx}]` etc.) so if in future they need to change it'll be a little easier (because they're named on the other end, rather than just being indexes).